ultimarc/devices/mini_pac.py

```diff
@@ -119,22 +120,66 @@ def _create_message_(self, config_file, cur_device_config=None):
             data.header.byte_3 = 0x0f
             data.header.byte_4 = header.asByte
 
-            # TODO: Allow macros to be optional and figure out macro naming functionality.
-            #  Does it have to be like the old way?
+            # key: Macro name value: macro value (e0 - fe)
+            macro_dict = {}
+            try:
+                # Macros
+                # Each macro starts with control character e0 - fe.  Total of 30 macros possible
+                # overall total macro characters is 85
+                max_size = 85
+                max_macro = 30
+                cur_size_count = 0
```

Comment:
'max_size' and 'max_macro' seem like they might be better as capitalized vars at the top of the file.

Reply:
I will make the change.  I figure they will end up in a base class when I add the next device.